README.md

```diff
@@ -115,6 +115,8 @@ standup --provider aws --context capa
 
 Flags:
       --cluster-values string         The path to the cluster app values
+      --release string                The version of Release to use to create the cluster (default "latest")
```

Comment:
"The version of Release" sounds weird in my brain :sweat_smile:

Reply:
Yeah, I dunno how else to describe it 🙈  Its the version of the Release CR to use.